SA3/qa_web/tests/test_views.py

```diff
@@ -246,7 +339,8 @@ def test_pagination(self):
         self.assertEqual(response.context['right'], range(4,6))
         self.assertEqual(response.context['latest_current_page'].number, 3)
```

Comment:
Wow I don't know we could test pagination like this. Impressive.

Reply:
Me neither. :)